.github/workflows/integration-tests.yml

```diff
@@ -111,6 +111,32 @@ jobs:
         run: ./test_tls.sh
         working-directory: ./tests/tls
 
+  test-tls-snapshot-shard-transfer:
```

Comment:
in a long run, can we really afford building custom CI task for individual tests?

Reply:
Should be fixed now.